.config/s3.config.php

```diff
@@ -14,6 +14,7 @@
         'region' => getenv('OBJECTSTORE_S3_REGION') ?: '',
         'hostname' => getenv('OBJECTSTORE_S3_HOST') ?: '',
         'port' => getenv('OBJECTSTORE_S3_PORT') ?: '',
+        'storageClass' => getenv('OBJECTSTORE_S3_STORAGE_CLASS') ?: 'STANDARD',
```

Comment:
Also note the unfortunately inconsistent naming convention. Soft reviews in upstream. This looks correct, going off of https://github.com/nextcloud/server/pull/36075/files#diff-dcd9e132a7c9faaa92d88afa9b90b299cc367bae128909d6c4c00f1c20952d94R50